03-getting-started/02-Configuration.adoc

```diff
@@ -10,101 +10,99 @@ toc::[]
+
+You can access configuration values via the **Config Provider**:
 
 [source, js]
 ----
 const Config = use('Config')
 const appSecret = Config.get('app.appSecret')
 ----
 
-The values are fetched using `Config.get()`. It takes as argument a string referencing the key you want, `fileName.key`. You can go deeper and fetch nested values using a dot-notation.
+Config values are fetched using `Config.get` which accepts a string argument referencing the key you want in the form `fileName.key`.
+
+You can fetch nested config values using dot notation:
 
 [source, js]
 ----
-// Example of a configuration - database.js
+// Example of a configuration file e.g. database.js
 {
   mysql: {
     host: '127.0.0.1',
   },
 }
 
-// You retrieve it using...
+// You can retrieve it like so...
 Config.get('database.mysql.host')
 ----
 
-If you aren't sure that the key is defined in your configuration you can supply a second argument that will be the default value.
+If you aren't sure that a key is defined in your configuration, you can supply a second argument that will be returned as the default value:
 
 [source, js]
 ----
 Config.get('database.mysql.host', '127.0.0.1')
 ----
 
-If you want to change the in-memory configuration values, you should make use of `Config.set()`.
+If you want to change the in-memory configuration values, use `Config.set`:
 
 [source, js]
 ----
 Config.set('database.mysql.host', 'db.example.com')
 ----
 
-WARNING: This only change the value in-memory. It will not write the new value on your config file.
+NOTE: `Config.set` will only change the value **in-memory**. It will not write the new value to your config file.
 
 == Env Provider
 
-When you are building an application, it happens quite often that you want to have a different configuration based upon the environment your code is running in.
+When building an application, you may want different configuration based upon the environment your code is running in.
 
-To fulfill this requirement, AdonisJs make use of the library link:https://github.com/motdotla/dotenv[dotenv, window="_blank"].
-Inside the root of every brand new AdonisJs project, you will find a `.env.example` file.
-If you used Adonis CLI to install your application, this file has been automatically copied to `.env`. Otherwise, you should copy it manually.
+To fulfill this requirement, AdonisJs uses the link:https://github.com/motdotla/dotenv[dotenv, window="_blank"] library.
 
-WARNING: The `.env` file should never be committed to your source control or shared with other people.
+Inside the root of every new AdonisJs project, you'll find an `.env.example` file.
+If you used AdonisJs CLI to install your application, this file will be automatically duplicated as `.env`. Otherwise, you should copy it manually.
 
-This file has a simple `key=value` syntax.
+NOTE: The `.env` file should never be committed to your source control or shared with other people.
```

Comment:
The reason I changed `WARNING:` to `NOTE:` is because there currently doesn't seem to be any `.warning` styles/scss in the [adonisjs/adonisjs.com](https://github.com/adonisjs/adonisjs.com) repository.

Because of this, if `WARNING:` is used, the box renders without colour or icon:

![warning](https://user-images.githubusercontent.com/1914481/47061637-0f290b00-d21e-11e8-9000-3f4acb6d45aa.png)

As you can see, using `NOTE:` at least renders the box correctly:

![note](https://user-images.githubusercontent.com/1914481/47061720-76df5600-d21e-11e8-9833-edc73a443ee8.png)

Guess I was thinking changing to `NOTE:` is a quick stylistic fix until someone implements `.warning` styles in the main repository?

Reply:
@furey Yeah let's go with the `warning` macro and I will meanwhile add the necessary CSS styles for it

Reply:
✅ https://github.com/adonisjs/docs/pull/314/commits/3798a4856d0f856afb8ec0c11622766c34fd5a39